src/Components/test/testassets/TestServer/Startup.cs

```diff
@@ -32,6 +32,13 @@ public void ConfigureServices(IServiceCollection services)
                     var detailedErrors = Configuration.GetValue<bool>("circuit-detailed-errors");
                     o.DetailedErrors = detailedErrors;
                 });
+            services.AddAuthentication(CookieAuthenticationDefaults.AuthenticationScheme).AddCookie();
+
+            services.AddAuthorization(options =>
+            {
+                options.AddPolicy("NameMustStartWithB", policy =>
+                    policy.RequireAssertion(ctx => ctx.User.Identity.Name?.StartsWith("B") ?? false));
+            });
```

Comment:
Why is this auth stuff going in here? Shouldn't it only be in the auth-specific startup class?

Reply:
Looks like you also removed it in a later commit. Correct?

Reply:
I took it out I believe